src/utils/__tests__/paths.spec.ts

```diff
@@ -46,6 +46,10 @@ describe('paths', () => {
       win32:  '/usr/local/bin',
       darwin: '/usr/local/bin',
     },
+    resources: {
+      win32:  new Error('does not apply'),
+      darwin: new Error('does not apply'),
```

Comment:
This seems very suspicious — why would that not apply? We definitely use this on both platforms.

(It looks like we never added Linux tests here; while we should do it, it wouldn't be this PR.)

Reply:
I confess: I was trying to get the PR in and got lazy, and didn't implement the changes to the unit tests I should have. Thanks for calling me on this. Working on it now.

Reply:
Oh, I was actually just surprised the tests _passed_… :D